src/solver/conv/conv_ocl_dir2D_bwdWrW_2.cpp

```diff
@@ -461,6 +463,18 @@ template <int N_BATCH_LOOPS>
 bool ConvOclBwdWrW2<N_BATCH_LOOPS>::IsApplicableBase(const ExecutionContext& ctx,
                                                      const ProblemDescription& problem) const
 {
+    // Disable this solver due to random GPU memory access faults on gfx11 and gfx12
+#if WORKAROUND_MIOPEN_1392
```

Comment:
```suggestion
#if WORKAROUND_LWPMIOPEN_1392
```

See the first comment